### PR TITLE
508 compliance bug fixes

### PR DESCRIPTION
**Pull request by dastan-rahimi**

### CODE REVIEW

File: LEAF_Nexus/js/nationalEmployeeSelector.js [hunk 1]
```diff
@@ -63,16 +63,17 @@ nationalEmployeeSelector.prototype.initialize = function() {
                     'VHA22',
                     'VHA23'];
     for(var i in tDomains) {
-    	domains += '<option value="'+ tDomains[i] +'">' + tDomains[i] + '</option>'
```
[the remark this answers is not in the document]
Comment (jeanmarcablevets) [reply LEAF_Nexus/js/nationalEmployeeSelector.js]: Addressed in commit 4de

File: LEAF_Request_Portal/templates/print_form_ajax.tpl [hunk 2]
```diff
@@ -3,7 +3,8 @@
 <div class="printmainform" style="border-bottom: 0px; min-height: 64px">
     <div id="requestTitle"><!--{$title|sanitize}--> <!--{$subtype|sanitize}-->
     <!--{if $submitted == 0 || $is_admin}-->
-        <img src="../libs/dynicons/?img=accessories-text-editor.svg&amp;w=16" style="cursor: pointer" alt="Edit Title" title="Edit Title" onclick="changeTitle()" />
+        <img src="../libs/dynicons/?img=accessories-text-editor.svg&amp;w=16" style="cursor: pointer" alt="Edit Title" title="Edit Title" onclick="changeTitle()" tabindex="0" role="button" onkeypress="if (event.keyCode==13){ changeTitle(); }" />
+=======
```
Comment (mgaoVA) [inline LEAF_Request_Portal/templates/print_form_ajax.tpl]: ?

File: LEAF_Request_Portal/templates/print_subindicators.tpl [hunk 3]
```diff
@@ -28,12 +28,12 @@
...
             <!--{/if}-->
             <!--{if $indicator.isWritable == 0}-->
-                <img src="../libs/dynicons/?img=emblem-readonly.svg&amp;w=16" alt="Read-only" title="Read-only" />
+                <img src="../libs/dynicons/?img=emblem-readonly.svg&amp;w=16" alt="Read-only" title="Read-only" tabindex="0" role="button" />
             <!--{else}-->
-                <img src="../libs/dynicons/?img=accessories-text-editor.svg&amp;w=16" alt="Edit this field" title="Edit this field" style="cursor: pointer" onclick="getForm(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->)" />
+                <img src="../libs/dynicons/?img=accessories-text-editor.svg&amp;w=16" alt="Edit <!--{$indicator.name|sanitizeRichtext|strip_tags}--> field" title="Edit <!--{$indicator.name|sanitizeRichtext|strip_tags}--> field" style="cursor: pointer" onclick="getForm(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->)" tabindex="0" role="button" onkeypress="if (event.keyCode==13){ getForm(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->); }" />
```
Comment (mgaoVA) [inline LEAF_Request_Portal/templates/print_subindicators.tpl]: In these context santizerRichText cannot be used. When used in alt-text, variables should be stripped of all tags.

File: LEAF_Nexus/templates/view_group.tpl [hunk 4]
```diff
@@ -33,9 +35,9 @@
 </div>
 
 <div id="toolbar" class="toolbar_right toolbar noprint">
-    <div id="tools"><h1>Options</h1>
+    <div id="tools"><h1 role="heading" tabindex="0">Options</h1>
```
Comment (mgaoVA) [inline LEAF_Nexus/templates/view_group.tpl]: Do labels like this need to be accessible via keyboard?

Comment (jeanmarcablevets) [reply LEAF_Nexus/templates/view_group.tpl]: Without the `tabindex`, the header isn't picked up by the screen reader.  The `role="heading"` tells the user that it is a header and not a button to be interacted with.

Comment (mgaoVA) [reply LEAF_Nexus/templates/view_group.tpl]: Ok, thanks.
